core/src/main/java/org/sonarsource/sonarlint/core/container/connected/update/RulesDownloader.java

```diff
@@ -49,13 +49,13 @@
   private static final org.sonarqube.ws.Rules.Active.Builder AR_BUILDER = Active.newBuilder();
   private static final org.sonarqube.ws.Rules.ActiveList.Builder ARL_BUILDER = ActiveList.newBuilder();
   private static final org.sonarqube.ws.Rules.Rule.Builder RULE_BUILDER = Rule.newBuilder();
-  private static final String RULES_SEARCH_URL = "/api/rules/search.protobuf?f=repo,name,severity,lang,htmlDesc,internalKey,isTemplate,templateKey,"
+  static final String RULES_SEARCH_URL = "/api/rules/search.protobuf?f=repo,name,severity,lang,htmlDesc,htmlNote,internalKey,isTemplate,templateKey,"
     + "actives&statuses=BETA,DEPRECATED,READY";
-  private static final String RULES_SEARCH_URL_MD_DESC = "/api/rules/search.protobuf?f=repo,name,severity,lang,htmlDesc,mdDesc,internalKey,isTemplate,templateKey,"
+  static final String RULES_SEARCH_URL_MD_DESC = "/api/rules/search.protobuf?f=repo,name,severity,lang,htmlDesc,mdDesc,htmlNote,mdNote,internalKey,isTemplate,templateKey,"
     + "actives&statuses=BETA,DEPRECATED,READY";
-  private static final String RULES_SEARCH_URL_JSON = "/api/rules/search?f=repo,name,severity,lang,htmlDesc,internalKey,isTemplate,templateKey,"
+  static final String RULES_SEARCH_URL_JSON = "/api/rules/search?f=repo,name,severity,lang,htmlDesc,htmlNote,internalKey,isTemplate,templateKey,"
     + "actives&statuses=BETA,DEPRECATED,READY";
-  private static final String RULES_SEARCH_URL_JSON_NO_DESC = "/api/rules/search?f=repo,name,severity,lang,internalKey,isTemplate,templateKey,"
+   static final String RULES_SEARCH_URL_JSON_NO_DESC = "/api/rules/search?f=repo,name,severity,lang,internalKey,isTemplate,templateKey,"
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)